src/pheval/utils/phenopacket_utils.py

```diff
@@ -37,6 +46,18 @@ def __str__(self):
 
 @dataclass
 class GenomicVariant:
+    """
+    Represents a genomic variant.
+
+    Args:
+        chrom (str): The chromosome position of the variant recommended to be provided in the following format.
+        This includes numerical designations from 1 to 22 representing autosomal chromosomes,
+        as well as the sex chromosomes X and Y, and the mitochondrial chromosome MT.
+        pos (int): Position of the variant
+        ref (str): Reference allele
+        alt (str): Alternate allele
```

Comment:
Is this a VCF-style pos, ref and alt? I'm going to assume so as there are a lot of combinations of these for representing variations. e.g. 1-start or 0-start (VCF uses 1-start), ref and alt alleles can be trimmed or untrimmed (use trimmed), trimming can be left or right (VCF uses left-trimmed), the ref and alt can either be present or absent, depending on how indels are represented. 

e.g. Clinvar variant [13294](https://www.ncbi.nlm.nih.gov/clinvar/variation/13294/) in VCF `10	123256215	.	T	G` a pathogenic missense variant (GRCh37 VCF coordinates - 1-based, positive strand) can be represented in SPDI and HGVS as shown below. It gets more complicated for indels:

Variant type | VCF | SPDI | HGVS
|--------------|-----|-------|-------|
SNP | 10	123256215	.	T	G | NC_000010.10:123256214:T:G | NC_000010.10:g.123256215T>G
DEL | 10	123256215	.	TC	T | NC_000010.10:123256215:C: | NC_000010.10:g.123256216del
INS | 10	123256215	.	T	TA | NC_000010.10:123256215::A | NC_000010.10:g.123256215_123256216insA 

To keep it clear and simple - just refer to the VCF spec.

Reply:
Sorry I’m confused by what you mean here. This is for extracting the data from the PhEval TSV results and from the Phenopacket variant for comparison in the analysis. Is this not over complicating things?

Reply:
I was taking this at face value - representing variants is something people have spent a huge number of hours (careers even?) debating, so just state that these follow VCF convention as tools will need to convert any output data they have into this format. If not stated, you have literally no idea what people will stuff in those fields.

Reply:
Got it! Done